### PR TITLE
RD-4629: Realignment of the form fields

### PR DESCRIPTION
**Pull request by NorbertAmartus**
## Description

Changes consists of:
- modifying form fields labels
- modifying the structure of the `formFields` object and extracting `ContactDetailsFormField` component to be able to easily adjust order and the allignment of the form fields
- adding explicit imports for the `lodash` functions

### Before
Before

### After
After

## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I verified that all tests and checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests

 

[Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/2924/)

## Documentation

N/A

### CODE REVIEW

File: app/translations/en.json [hunk 1]
```diff
@@ -30,26 +30,26 @@
         "form": {
             "fields": {
                 "firstName": {
-                    "label": "First name",
+                    "label": "First Name",
```
Comment (kubama) [inline app/translations/en.json:33]: Why the change to uppercase 'N'?

Comment (NorbertAmartus) [reply app/translations/en.json:33]: Please take a look at the screenshot attached to [RD-4629](https://cloudifysource.atlassian.net/browse/RD-4629)
Here we can see, that form contains a field with the `First Name` label, not `First name`

And as the ticket is about alligning labels / fields layout accordingly to the provided screenshot, I've put here an uppercase 'N'

Comment (kubama) [reply app/translations/en.json:33]: I see no reason why it should be in upper case. Besides, why 'Phone **n**umber' but 'First **N**ame'? Personally I would consult it with ticket submitter, but since it's a minor thing and it's clearly stated in the ticket I have no other option than to approve :slightly_smiling_face:.
